src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -281,6 +282,9 @@ def lsm_service_catalog_get_entity(
         This is a mock for the lsm api, this method is called during export of the
         service entities.
         """
+        assert self.service_entities is not None, "Bad usage!!"
```

Comment:
The assert message could give some suggestions on the correct usage

Reply:
Indeed, fixed